README.md

```diff
@@ -47,10 +53,15 @@ but all the dependency modules will remain in the system.
 
 ## Project contents
 
-`__main__.py` is project executable. It uses the following modules from `universum` directory:
-* `main`/`poll`/`submit`/`api` - managing modules for different Universum modes
+`universum` is main project folder, that is being copied to `dist_packages` when installed.
```

Comment:
On my system it is copied to site-packages.
dist-packages (note the hyphen) are used for storing packages installed by OS distribution.